assets/lambda/code/scan/lambda.py

```diff
@@ -128,6 +137,53 @@ def set_status(bucket, key, status):
     metrics.add_metric(name=status, unit="Count", value=1)
 
 
+def get_status(bucket_name, key):
+    """
+    Retrieve the value of the status tag from an S3 object
+
+    Parameters:
+        bucket_name (str): Name of the S3 bucket.
+        object_key (str): The key (path) of the S3 object.
+    """
+    return get_tag_value(bucket_name, key, "scan-status')
```

Comment:
Fix name of key to object_key

```suggestion
def get_status(bucket_name, object_key):
    """
    Retrieve the value of the status tag from an S3 object

    Parameters:
        bucket_name (str): Name of the S3 bucket.
        object_key (str): The key (path) of the S3 object.
    """
    return get_tag_value(bucket_name, object_key, "scan-status')
```